test/spec/app_spec.rb

```diff
@@ -1066,21 +1066,98 @@
       expect(current > 0).to be(true)
     end
 
-    # it "should find medias when searching by keyword" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should find medias when searching by keyword", bin2: true do
+      data = api_create_team_and_project
+      api_create_media(data: data, url: "https://www.facebook.com/permalink.php?story_fbid=10155901893214439&id=54421674438")
+      media = api_create_media(data: data, url: "https://twitter.com/TwitterVideo/status/931930009450795009")
+      @driver.navigate.to media.full_url
```

Comment:
This step is useless, no?

Reply:
It is there due the "get_team" in the next line.
get_team updated to data[:team].slug and step removed.